### DISCUSSION TITLE
How to set a custom statusText (using context ) for a https response.

### DISCUSSION THREAD

**Discussion by HayatoKazi** [honojs/hono#2630] [Q&A]
For an API endpoint , I am able to set status for a HTTP request, however while checking on the response, I could find statusText value as empty string. 

Is there any way I can provide statusText response using context object.?

**Comment by NicoPlyley** [comment]
I believe responseText has been considered deprecated in web standards while using fetch (which is what Hono is built of) because it was unreliable. Now we reply on a the `response.ok` boolean. I believe that is why it is not incorporated in Hono. But maybe @yusukebe can confirm this

**Comment by yusukebe** [comment]
@tsdineshjai 

> Is there any way I can provide statusText response using context object.?

There is no way. So, you have to use `c.body()` or directly `new Response()`:

```ts
return c.body('abc', {
  status: 200,
  statusText: 'foo'
})
```

```ts
return new Response('abc', {
  status: 200,
  statusText: 'foo'
})
```
